production/helm/loki/templates/_helpers.tpl

```diff
@@ -359,3 +359,14 @@ Create the service endpoint including port for MinIO.
 {{- define "enterprise-logs.canarySecret" }}
 {{- .Values.enterprise.canarySecret | default (printf "%s-canary-secret" (include "loki.name" . )) -}}
 {{- end -}}
+
+{{/* Configure enableServiceLinks in pod */}}
+{{- define "loki.enableServiceLinks" -}}
+{{- if semverCompare ">=1.13-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
why `KubeVersion.GitVersion` instead of `KubeVersion.Version`. We are currently using the latter, and I'd like to keep in consistent if possible.

Reply:
That was helm v2, let me change to `KubeVersion.Version` instead